language/de-DE/com_users.ini

```
@@ -108,11 +139,25 @@ COM_USERS_RESET_REQUEST_ERROR="Fehler bei der Anfrage zum Zurücksetzen des Pass
 COM_USERS_RESET_REQUEST_FAILED="Zurücksetzen des Passworts fehlgeschlagen: %s"
 COM_USERS_RESET_REQUEST_LABEL="Bitte die E-Mail-Adresse des Benutzerkontos eingeben. Ein Bestätigungscode wird dann an diese verschickt. Sobald der Code vorliegt, kann ein neues Passwort für das Benutzerkonto festgelegt werden."
 COM_USERS_SETTINGS_FIELDSET_LABEL="Basiseinstellungen"
+COM_USERS_USER_BACKUPCODE="Backup-Code"
+COM_USERS_USER_BACKUPCODES="Backup-Codes"
+COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="Wenn kein Zugriff mehr auf die übliche Multi-Faktor-Authentifizierung Methode besteht, kann einer der Backup-Codes im folgenden Feld verwendet werden. Dieser Backup-Code kann dann kein weiteres Mal mehr verwendet werden."
```

```suggestion
COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="Falls die Anmeldung mit gewohnten Multi-Faktor-Authentifizierungsmethode nicht funktioniert, kann einer der Backup-Codes in das untenstehende Feld eingegeben werden. Achtung: Dieser Notfall-Backup-Code kann nicht erneut verwendet werden."
```

---

Haben wir noch einen Mix aus Notfall-Einmalpasswort und Notfall-Backup-Code, Backup-Code?

---

```suggestion
COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="Falls die Anmeldung mit gewohnten Multi-Faktor-Authentifizierungsmethode nicht funktioniert, kann einer der Backup-Codes in das untenstehende Feld eingegeben werden. Achtung: Dieser Backup-Code für den Notfall kann nicht erneut verwendet werden."
```

Wie wäre diese Alternative?

---

> Haben wir noch einen Mix aus Notfall-Einmalpasswort und Notfall-Backup-Code, Backup-Code?

Wo taucht "Notfall-Einmalpasswort" auf?